src/components/ResourceList/components/Item/Item.tsx

```diff
@@ -334,6 +335,10 @@ export class Item extends React.PureComponent<CombinedProps, State> {
       onClick(id);
     }
 
+    if (ctrlKey || metaKey) {
+      return window.open(url, '_blank');
```

Comment:
Should do a check for `url` because it's not required if there's an onClick.

Also I wonder, if there is an onClick and a URL should the this command click happen first and the onClick be ignored?

Reply:
I figured we should err on the side of caution here and allow the onClick to fire regardless.